src/app/pages/home/home.component.css

```
@@ -319,3 +233,41 @@ markdown {
   height: 50px;
   margin: -50px 0 0 0;
 }
+
+
+/* || Animations */
+@-webkit-keyframes SectionGradientAnimation {
+  0% {
+    background-position: 0 0
+  }
```

```suggestion
  0% { background-position: 0 0 }
```
Could you do this for the other ones too? I think this would be useful to make the code easier to read.